migrations/2_deploy_contracts.js

```diff
@@ -51,10 +57,26 @@ module.exports = deployer =>
     );
     const publicCommitment = await PublicCommitment.deployed();
     console.log("Commitment deployed");
-    await lock.setController(publicCommitment.address);
+    console.log("Seting permissions");
+    await deployer.deploy(AccessRoles);
+    const accessRoles = await AccessRoles.deployed();
+    await accessControl.setUserRole(
+      accounts[1],
+      await accessRoles.ROLE_LOCKED_ACCOUNT_ADMIN(),
+      lock.address,
+      1
+    ); // 1 is True
```

Comment:
I would put a `const TriState = { Unset:0, Allowed: 1, Denied: 2};` at the top of this file and use that. Than we don't need the comment, don't have magic numbers and can change it in one location.

Even nicer would be to get the enum values directly out of the contract, but that may not be worth the effort.

Reply:
👍 for extracting magic numbers to top of the file